extensions/mail/src/mark-all-as-read.tsx

```diff
@@ -0,0 +1,24 @@
+import { Toast, showHUD, showToast } from "@raycast/api";
+import { runAppleScript } from "run-applescript";
+
+async function script(): Promise<void> {
+  const script = `
+    tell application "Mail"
+      set visible of window 1 to false 
+      set read status of every message of every mailbox of every account where its read status = false to true
+      set visible of window 1 to true
+    end tell
+`;
+
+  runAppleScript(script);
+}
+
+export default async function command() {
```

Comment:
In order to keep it consistent, can you rename this to `MarkAllAsRead`?

Reply:
Hey @ikupenov and @pernielsentikaer, thanks for the feedback. 
I just pushed a new commit with the changes.